README.md

```diff
@@ -1,64 +1,23 @@
-
-## Build automation using Gradle
-
-* This project uses Gradle for build automation and dependency management. It includes a basic build script as well (i.e. the `build.gradle` file).
-* If you are new to Gradle, refer to the [Gradle Tutorial at se-education.org/guides](https://se-education.org/guides/tutorials/gradle.html).
-
-## Testing
-
-### I/O redirection tests
-
-* To run _I/O redirection_ tests (aka _Text UI tests_), navigate to the `text-ui-test` and run the `runtest(.bat/.sh)` script.
-
-### JUnit tests
-
-* A skeleton JUnit test (`src/test/java/seedu/duke/DukeTest.java`) is provided with this project template. 
-* If you are new to JUnit, refer to the [JUnit Tutorial at se-education.org/guides](https://se-education.org/guides/tutorials/junit.html).
-
-## Checkstyle
-
-* A sample CheckStyle rule configuration is provided in this project.
-* If you are new to Checkstyle, refer to the [Checkstyle Tutorial at se-education.org/guides](https://se-education.org/guides/tutorials/checkstyle.html).
-
-## CI using GitHub Actions
-
-The project uses [GitHub actions](https://github.com/features/actions) for CI. When you push a commit to this repo or PR against it, GitHub actions will run automatically to build and verify the code as updated by the commit/PR.
-
-## Documentation
-
-`/docs` folder contains a skeleton version of the project documentation.
-
-Steps for publishing documentation to the public: 
-1. If you are using this project template for an individual project, go your fork on GitHub.<br>
-   If you are using this project template for a team project, go to the team fork on GitHub.
-1. Click on the `settings` tab.
-1. Scroll down to the `GitHub Pages` section.
-1. Set the `source` as `master branch /docs folder`.
-1. Optionally, use the `choose a theme` button to choose a theme for your documentation.
+ <br />
+ <div align="center">
+    <a href="">
+        <img src="docs/images/ChessMasterLogo.png" alt="Logo" width="80" height="80">
+    </a>
+    <h3 align="center">ChessMaster</h3>
+    <p align="center">
+        Chess finally makes it to your terminal.
+        <br />
+        <a href="https://ay2324s1-cs2113-t18-1.github.io/tp/UserGuide.html">View User Guide</a>
+        ·
+        <a href="https://ay2324s1-cs2113-t18-1.github.io/tp/DeveloperGuide.html">View Developer Guide</a>
+        ·
+        <a href="https://ay2324s1-cs2113-t18-1.github.io/tp/AboutUs.html">About Us</a>
+    </p>
+ </div>
+
+_ChessMaster_ is a command-line interface (CLI) chess game designed to make learning and training accessible for
+beginners while offering an engaging experience for all skill levels. This simple and user-friendly chess simulator
+provides a platform for novice players to build their skills and understanding of the game.
+
+This project was created for the _CS2113: Software Engineering and Object-Oriented Programming_ team project and
+is programmed in 100% Java.
```

Comment:
Perhaps consider adding basic installation instructions? Like copy paste the quickstart guide from the UG

Reply:
I've gone ahead and added these now, along with a link to the UG.